types/wordpress__blocks/index.d.ts

```diff
@@ -503,3 +504,20 @@ export type Transform<T extends Record<string, any> = Record<string, any>> =
     | TransformPrefix<T>
     | TransformRaw<T>
     | TransformShortcode<T>;
+
+export type BlockVariationScope = 'block' | 'inserter' | 'transform';
+
+export interface BlockVariation<T extends Record<string, any> = any> {
```

Comment:
the `extends Record<string, any>` is fine but the default to `any` feels suspect. do we need to do that or could we say that the default in case no attributes are provided is `{}`, more specifically `Record<string, any>`?

```ts
export interface BlockVariation<Attributes extends Record<string, any> = Record<string, any>> {
```

with an alias

```ts
interface BlockAttributes extends Record<string, any> {}

export interface BlockVariation<Attributes extends BlockAttributes = BlockAttributes> {
```

Reply:
Good idea and used it for the mentioned InnerBlockConfiguration.